### PR TITLE
Creation of popup modals for project details

### PR DESCRIPTION
**Pull request by redwing555**
This PR is for the Popup modals for details about my projects in my personal portfolio webpage:

⏩ Populating Html Recent work section using JavaScript.
⏩ Creation of Popup windows when the "see Project" button is clicked using JavaScript.
⏩Blurring of the background of the Popup windows.
⏩ Respect of Figma design as well as UX design, HTML, CSS, and JavaScript best practices.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -95,176 +103,31 @@ <h1 class="recent-works">My Recent Works</h1>
...
-          <div>
-            <h2>
-              Multi-Post Stories<br />
-              Gain+Glory
-            </h2>
-          </div>
+ 
+     
+    
+    <div class="wrapper"  id="portfolio" >
```
Comment (DanteAlonsoHT) [inline index.html:109]: All your "Recent works" are missing in your website, you should display them dynamically using JavaScript, kindly, fix it.

image

File: index.js [hunk 2]
```diff
@@ -1,13 +1,222 @@
...
-
 const menu = document.querySelector('.hide_mobile');
+const projectCards = document.querySelector('.wrapper');
+const contactContain = document.getElementById('contact');
+const aboutContain = document.getElementById('aboutme');
+const mainContain = document.getElementById('portfolio');
+/* const popupContain = document.querySelectorAll('popup-window'); */
+const homepageContain = document.getElementById('homepage');
+// const recentworkContain = document.querySelector("recent-work");
+
```
Comment (DanteAlonsoHT) [inline index.js:12]: - [ ] [OPTIONAL] Remove all your comments ( // ) in your JavaScript file to make your code more impressive 😉

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @redwing555, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Follow the Figma design you selected, there are many styles you can implement in your whole website.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mke2111 — APPROVED** [review]
### Hi @redwing555, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉** 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**